.github/workflows/docs.yml

```
@@ -0,0 +1,50 @@
+name: Docs
+
+on:
+  push:
+    paths:
+      - ".github/workflows/docs.yml"
+      - "docs/**"
+  pull_request:
+    paths:
+      - ".github/workflows/docs.yml"
+      - "docs/**"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    name: Docs
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+        cache: pip
+        cache-dependency-path: ".ci/*.sh"
+
+    - name: Build system information
+      run: python3 .github/workflows/system-info.py
+
+    - name: Install Linux dependencies
+      run: |
+        .ci/install.sh
```

```suggestion
        .ci/install.sh
      env:
        GHA_PYTHON_VERSION: "3.x"
```
I know it's not a specific Python version, but for the purposes of
https://github.com/python-pillow/Pillow/blob/eccc1e9afb4b2b012772abdad026e1efa49b0e9f/.ci/install.sh#L41-L44
it is sufficient.

---

I left it out because I don't think we specifically need PyQt for the docs build? Well, let's use the env var so as not to worry about it in the future. 👍